data/styles/clear/include/Icons.mapcss

```diff
@@ -2449,6 +2449,9 @@ area|z16[shop=greengrocer]
 node|z16[shop=hairdresser],
 area|z16[shop=hairdresser]
 {icon-image: hairdresser-m.svg;}
+node|z16[shop=sewing],
+area|z16[shop=sewing]
+{icon-image: needle_and_thread-m.svg;}
```

Comment:
should be `z16-` so that it displays on all higher zoom levels. currently it _only_ displays on z16 :)